src/main/java/seedu/address/model/person/Email.java

```diff
@@ -8,6 +8,10 @@
  * Guarantees: immutable; is valid as declared in {@link #isValidEmail(String)}
  */
 public class Email {
+    //Default Email if no email is specified
+    public static final String EMPTY_EMAIL = "";
```

Comment:
Rather than having an EMPTY_EMAIL field, you could consider just using the Optional class to encapsulate the email instead, same for Address, if you would like.

Reply:
I tried to change the email and address field in the Person's attribute to Optional<Email> and Optional<Address>, but it require changes in a lot of files as now emails and addresses are encapsulated in Optional (especially in tests). As we are planning to add more optional fields, I will consider this once I tried to implement those new optional fields by using Optional<>. Is that ok?

Reply:
Oh yes that is true, nevermind then, I think we can merge this for now.